runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -146,8 +162,13 @@ public Void call() {
             Optional<Counter> conflictCounter = MicroMeterUtils.counter("logreplication.subscribe.conflict.count");
             Optional<Timer.Sample> subscribeTimer = MicroMeterUtils.startTimer();
 
-            Table<LogReplicationSession, ReplicationStatus, Message> replicationStatusTable =
-                    openReplicationStatusTable(corfuStore);
+            try {
+                replicationStatusTable = corfuStore.getTable(CORFU_SYSTEM_NAMESPACE, REPLICATION_STATUS_TABLE_NAME);
+            } catch (NoSuchElementException | IllegalArgumentException e) {
+                log.warn("Failed getTable operation for LR status table, opening table.", e);
+                replicationStatusTable = openReplicationStatusTable(corfuStore);
```

Comment:
Will we need a retry here as well as a try - catch block for the openReplicationStatusTable?

Reply:
Yes that would be good have so all failures from this task can be handled by the listeners onError, added.